src/main/webapp/admin_dashboard.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Admin Dashboard</title>
+		<link rel="stylesheet" href="admin_dashboard_style.css">
+		<script src='https://kit.fontawesome.com/a076d05399.js'></script>
+        <script src="https://www.gstatic.com/charts/loader.js"></script>
+        <script src="admin_dashboard_script.js"></script>
+	</head>    
+	<body onresize="drawChart()">
+		<div class="wrapper">
+			<nav>
+				<header>NeighborGood</header>
+				<ul>
+					<li><a class="active">Dashboard</a></li>
+					<li><a>My Account</a></li>
+					<li><a>Statistics</a></li>
+					<li><a>Settings</a></li>
+					<li><a>Logout</a></li>
+				</ul>
+			</nav>
+
+			<main>
+				<h1>Admin Prototype</h1>
+				<div class="notification-grid">
```

Comment:
We might want to think whether we should remove this for the prototype and re-add it if we end up having a notification system. My understanding is that the prototype includes what we are going to implement, but I could be wrong!